src/http/reply.cc

```diff
@@ -49,144 +50,68 @@ namespace http {
+    {reply::status_type::created, "201 Created"},
+    {reply::status_type::accepted, "202 Accepted"},
+    {reply::status_type::nonauthoritative_information, "203 Non-Authoritative Information"},
+    {reply::status_type::no_content, "204 No Content"},
+    {reply::status_type::reset_content, "205 Reset Content"},
+    {reply::status_type::partial_content, "206 Partial Content"},
+    {reply::status_type::multiple_choices, "300 Multiple Choices"},
+    {reply::status_type::moved_permanently, "301 Moved Permanently"},
+    {reply::status_type::moved_temporarily, "302 Moved Temporarily"},
+    {reply::status_type::see_other, "303 See Other"},
+    {reply::status_type::not_modified, "304 Not Modified"},
+    {reply::status_type::use_proxy, "305 Use Proxy"},
+    {reply::status_type::temporary_redirect, "307 Temporary Redirect"},
+    {reply::status_type::bad_request, "400 Bad Request"},
+    {reply::status_type::unauthorized, "401 Unauthorized"},
+    {reply::status_type::payment_required, "402 Payment Required"},
+    {reply::status_type::forbidden, "403 Forbidden"},
+    {reply::status_type::not_found, "404 Not Found"},
+    {reply::status_type::method_not_allowed, "405 Method Not Allowed"},
+    {reply::status_type::not_acceptable, "406 Not Acceptable"},
+    {reply::status_type::request_timeout, "408 Request Timeout"},
+    {reply::status_type::conflict, "409 Conflict"},
+    {reply::status_type::gone, "410 Gone"},
+    {reply::status_type::length_required, "411 Length Required"},
+    {reply::status_type::payload_too_large, "413 Payload Too Large"},
+    {reply::status_type::uri_too_long, "414 URI Too Long"},
+    {reply::status_type::unsupported_media_type, "415 Unsupported Media Type"},
+    {reply::status_type::expectation_failed, "417 Expectation Failed"},
+    {reply::status_type::page_expired, "419 Page Expired"},
+    {reply::status_type::unprocessable_entity, "422 Unprocessable Entity"},
+    {reply::status_type::upgrade_required, "426 Upgrade Required"},
+    {reply::status_type::too_many_requests, "429 Too Many Requests"},
+    {reply::status_type::login_timeout, "440 Login Timeout"},
+    {reply::status_type::internal_server_error, "500 Internal Server Error"},
+    {reply::status_type::not_implemented, "501 Not Implemented"},
+    {reply::status_type::bad_gateway, "502 Bad Gateway"},
+    {reply::status_type::service_unavailable, "503 Service Unavailable"},
+    {reply::status_type::gateway_timeout, "504 Gateway Timeout"},
+    {reply::status_type::http_version_not_supported, "505 HTTP Version Not Supported"},
+    {reply::status_type::insufficient_storage, "507 Insufficient Storage"},
+    {reply::status_type::bandwidth_limit_exceeded, "509 Bandwidth Limit Exceeded"},
+    {reply::status_type::network_read_timeout, "598 Network Read Timeout"},
+    {reply::status_type::network_connect_timeout, "599 Network Connect Timeout"}};
+
+static const std::string_view& to_string_view(reply::status_type status) {
```

Comment:
// since Pavel is reviewing.. 

how about defining a `fmt::formatter` for `status_type`?

Reply:
Dunno... Who gonna use it? All this string manipulation is local to this file only, no one never used it outside.